src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -576,4 +576,19 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="ListPkg_InvalidOptionsOutdatedAndDeprecated" xml:space="preserve">
     <value>Invalid command. Combining the '--outdated' and '--deprecated' options is not supported.</value>
   </data>
+  <data name="ListPkg_NoDeprecatedPackagesForProject" xml:space="preserve">
+    <value>The given project `{0}` has no deprecated packages given the current sources.</value>
+  </data>
+  <data name="ListPkg_ProjectDeprecationsHeaderLog" xml:space="preserve">
+    <value>Project `{0}` has the following deprecated packages</value>
```

Comment:
Do we need any punctuation here?

Reply:
This is a header, followed same punctuation (none) as used in [`ListPkg_ProjectUpdatesHeaderLog`](https://github.com/NuGet/NuGet.Client/pull/2959/files#diff-bf848a6e817170bb1de10d240cf12fcdR571).